data/avgi/avgi 1 introduction.txt

```diff
@@ -0,0 +1,879 @@
+			`	Prasinos says, more seriously, "Reservation. Our material resources are quite limited at the moment, so for now I would only support providing for the requisition of small items that do not strain the needs of the Guard. No military goods, at least not now."`
+			`	Asime looks like she wants to object, but then Galano says, "Agreement. A reasonable compromise, one I can sell to the Assembly fairly easily. Beyond that, we will provide you with a line of credit with which to purchase housing and nutrition. I cannot vouch for its usefulness among... certain elements that do not recognize the Consonance, but anywhere else, you should find no issues."`
+			action
+				set "license: Avgi Civilian"
+			`	"Conclusion. For now, we will leave you with some time and space to think. The former Captain of the ship that brought the other humans, one Darius Calwell, is currently advising construction on the cylinder habitat in this system. You will find him on Aktina Cylinder, if you wish to speak with him."`
+			`	You are dismissed, and on the way out you are given a black hairpin-sized translator device, a smaller cousin of the one the Avgi have been using to communicate with you.`
+			action
+				set "language: Avgi"
+
+mission "Avgi: Humans From the Deep"
+	name "Humans Among the Avgi"
+	description "Visit the <destination>, where there is another human, Darius Calwell, stranded in Avgi space."
+	source "Peripheria"
+	landing
+	destination "Aktina Cylinder"
+	to offer
+		has "Avgi: First Contact: done"
+
+
+mission "Avgi: Defend Ensemble 1"
+	name "Avgi Strike Force"
+	description "Meet up with the Avgi strike force at <destination>."
+	source "Aktina Cylinder"
+	landing
+	passengers 1
+	blocked "You'll need a free bunk on the <ship> if you want to take Darius anywhere off this station."
+	destination "Napigeo Station"
+	to offer
+		has "Avgi: Humans From the Deep: done"
+	on offer
+		conversation
+			`From afar, Aktina station is a titanic skeleton, with graphene ribs and a metallic backbone. Reminiscent of the human concept of an O'Neil Cylinder, it will be able to house millions of Avgi once completed. For now though, its construction has been hampered by the constant demand for new warships and mining stations throughout the Tangled Shroud, to the point that many are skeptical of its projected completion time within another few years.`
+			`	Following the instructions of the station's control authority, you fly through the station's open frame to its small pressurized area, which houses the construction workers and maintenance crews. You've been told that word of your arrival has been sent ahead of you, so you aren't entirely surprised to see a middle-aged human waiting in the hangar, a look of pure astonishment on his face as your ship settles into a free space. You disembark the <ship> to find Darius waiting right outside the hatch.`
+			choice
+				`	"Hello."`
+				`	"Hey there."`
+				`	"I come in peace."`
```

Comment:
If you want to emphasise expression, then there should be different responses to the choices.

As for the choices, I don't really register "Hey there" as that much more casual than "Hello", and "I come in peace" sounds to me more like a really polite non-native english speaker rather than an active attempt to be funny.